nmosnode/nodefacadeservice.py

```diff
@@ -18,33 +18,33 @@
 from gevent import monkey
 monkey.patch_all()
 
-import time
-import signal
-import os
-import sys
-import json
-
-from nmoscommon.httpserver import HttpServer
-from nmoscommon.utils import get_node_id
-from socket import gethostname, getfqdn
-from .api import FacadeAPI
-from .registry import FacadeRegistry, FacadeRegistryCleaner, legalise_resource
-from .serviceinterface import FacadeInterface
-from os import getpid
-from subprocess import check_output
-from systemd import daemon
-
-from .api import NODE_APIVERSIONS
-from .api import NODE_REGVERSION
-
-from nmoscommon.utils import getLocalIP
-from nmoscommon.aggregator import Aggregator
-from nmoscommon.aggregator import MDNSUpdater
-from nmoscommon.mdns import MDNSEngine
-from nmoscommon.logger import Logger
-from nmoscommon import ptptime
-from nmoscommon.nmoscommonconfig import config as _config
-import socket
+import time # noqa E402
```

Comment:
I'm not keen on this, but it doesn't seem there's really a better alternative :(

Reply:
Yeah same but its either that or turn it off for the repo globally